src/style.css

```diff
@@ -0,0 +1,85 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+.bigsearch {
+  height: 70px;
+}
+
+.smallsearch {
+  height: 42px;
+}
+
+.search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-color: #fff;
+  margin: 20px auto;
+  display: flex;
+  box-sizing: border-box;
+  color: #3d4e61;
+  font-family: Avenir;
+  font-weight: 500;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus {
+  outline: none;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.input {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.inputsmall {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+::placeholder {
```

Comment:
Better not use attribute selector. Use class one

Reply:
It was the part of the task, that we can add styles through pseudo-element.

Reply:
I think Viktor means you should to use pseudo-element like `.input::placeholder` (example)